pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -161,6 +163,44 @@ def getLevels(self):
         When ``autoLevels`` is active, the format is [`blackLevel`, `whiteLevel`].
         """
         return self.levels
+        
+    def setColorMap(self, colorMap):
+        """
+        Sets a color map for false color display of a monochrome image.
+
+        Parameters
+        ----------
+        colorMap : `pyqtgraph.ColorMap` or `str`
+            A string argument will be passed to `pyqtgraph.colormap.get()`
+        """
+        if isinstance(colorMap, colormap.ColorMap):
+            self._colorMap = colorMap
+        elif isinstance(colorMap, str):
+            self._colorMap = colormap.get(colorMap)
+        else:
+            raise TypeError("'colorMap' argument must be ColorMap or string")
+        self.setLookupTable( self._colorMap.getLookupTable(nPts=256) )
+        
+    def getColorMap(self):
+        """
+        Returns the assigned color map, or None if not available
+        """
+        return self._colorMap
+        
+    # def addColorBar(self, colorMap=None, plot=None, **kargs):
```

Comment:
Looks like just need to delete this commented out code and this is probably good to merge? I should take this diff for a spin myself first I suppose.